### PR TITLE
Change how TBFighters socials are formatted and add a link to the newsletter

### PR DESCRIPTION
**Pull request by DFTBA-Shalom**
# Description
Moved the social media accounts at the top of the action page to their own paragraph, messed with the copy a bit, and added a link to the newsletter as another source of information and updates.
I also moved the socials icons in the middle of the action page to right above the newsletter, since I think it makes more sense to avoid breaking the flow of resources related to directly engaging with action and that the newsletter and socials are more related than the stuff around it previously.

# Type of change
- [ ] Correction (typos, outdated links, etc.)
- [ ] Updated content (New CTA, updated wording, etc.)

# Misc Notes
Please don't merge this immediately! I'd like to offer some time for discussion in web dev, especially around the copy changes.

### CODE REVIEW

File: action.html [hunk 1]
```diff
@@ -275,6 +268,19 @@ <h4 id="graphics">And some of our most popular <a href="https://tbfighters.notio
...
+					<h2 id="socials"><span class="red">Follow</span> us on:
+					<div class="header-copy"><noscript><a href="#posts"></a></noscript></div>
+					</h2>
+					<p class="larger socials-wrapper">
+						<a href="https://x.com/tbfighters" target="_blank"><img src="img/socials/twitter.svg" alt="Twitter" class="social-icon"><img src="img/socials/twitter-hover.svg" alt="" class="social-icon-hover"></a>
+						<a href="https://www.instagram.com/tbfighters" target="_blank"><img src="img/socials/insta.svg" alt="Instagram" class="social-icon"><img src="img/socials/insta-hover.svg" alt="" class="social-icon-hover"></a>
+						<a href="https://www.youtube.com/@TB_Fighters" target="_blank"><img src="img/socials/youtube.svg" alt="Youtube" class="social-icon"><img src="img/socials/youtube-hover.svg" alt="" class="social-icon-hover"></a>
+						<a href="https://www.facebook.com/profile.php?id=61555332203505" target="_blank"><img src="img/socials/facebook.svg" alt="Facebook" class="social-icon"><img src="img/socials/facebook-hover.svg" alt="" class="social-icon-hover"></a>
+					</p>
+
```
Comment (YummyOreo) [inline action.html:283]: you forgot a </div> here!

File: action.html [hunk 2]
```diff
@@ -275,6 +268,19 @@ <h4 id="graphics">And some of our most popular <a href="https://tbfighters.notio
 						</noscript>
 					</div>
 				</div>
+				
+				<div class="wrapper-full" style="transition: ease 0.5s all;">
```
Comment (YummyOreo) [inline action.html:272]: the `style="transition: ease 0.5s all;"` doesn't need to be here

File: action.html (hunk 1 above)
Comment (YummyOreo) [reply action.html:283]: on line 283, it will fix the indenting problem

Comment (YummyOreo) [reply action.html:283]:
```suggestion
					</div>
```

**Review by YummyOreo — APPROVED** [review]

File: action.html (hunk 2 above)
Comment (DFTBA-Shalom) [reply action.html]: Thank you!!

**Review by Nolram12345 — APPROVED** [review]

**Pull request merged**